doc/src/user_guide.rst

```diff
@@ -39,6 +39,16 @@ or greater. The backend requires:
 1. `CUDA <https://developer.nvidia.com/cuda-downloads>`_ >= 4.2
 2. `pycuda <http://mathema.tician.de/software/pycuda/>`_ >= 2011.2
 
+MIC Backend
+^^^^^^^^^^^
+
+The MIC backend targets Intel Xeon Phi co-processors. The backend
+requires:
+
+1. ICC >= 16.0
```

Comment:
I found that the recent intel MPSS 3.6 (system level software for Xeon Phi) has a problem to build pyMIC. MPSS 3.3, 3.4 or 3.5 don't have a problem. I will report this to pyMIC soon. I confirmed that the recent icc and mkl, from Intel(R) Composer XE 2013 SP1, 2015 or 2016, work well.